ci/docker/build/Dockerfile

```diff
@@ -8,19 +8,30 @@ ENV PATH $PATH:$DEPOT_TOOLS_PATH
 # Notes:
 # - libx11-dev is used by Flutter for desktop linux (see also install-build-deps-linux-desktop.sh)
 # - chrome is used by Flutter for web.
+
 RUN apt-get update && \
     apt-get install -y git wget curl unzip python lsb-release sudo apt-transport-https && \
     git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git $DEPOT_TOOLS_PATH && \
     mkdir --parents $ENGINE_PATH && \
     git clone https://github.com/flutter/buildroot.git $BUILDROOT_PATH && \
     cd $BUILDROOT_PATH && \
-    ./build/install-build-deps.sh --no-prompt && \
-    ./build/install-build-deps-android.sh --no-prompt && \
-    (echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list) && \
-    (curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -) && \
-    (wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -) && \
-    echo 'deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main' | tee /etc/apt/sources.list.d/google-chrome.list && \
-    apt-get update && apt-get install -y google-cloud-sdk google-chrome-stable libx11-dev && \
+    apt-get install -y autoconf automake bison flex gettext gperf groff libtool pkg-config acpica-tools build-essential dejagnu \
+      dosfstools ifupdown libedit-dev libglib2.0-dev liblz4-tool libncurses5-dev libssl-dev lsof mtools nasm net-tools python2.7-dev \
+      python-m2crypto tcpdump texinfo uuid-dev vim xz-utils zlib1g-dev firefox-esr g++-multilib libblkid-dev libc6-dev-i386 \
+      libfreetype6-dev libegl1-mesa libgles2-mesa-dev libglu1-mesa-dev libgtk-3-dev libx11-dev libxcursor-dev libxinerama-dev libxrandr-dev \
+      libxxf86vm-dev mesa-common-dev openjdk-8-jdk zip xvfb ca-certificates gnupg
```

Comment:
Not for this patch, but out of curiosity does this all need to be a single `RUN` command? 

I wonder if it would be easier to follow if we broke into sections (in a followup patch) like:
1. apt-get update
2. Install gpg to support adding repos
3. Add additional repos like chrome, gcloud
4. apt-get update
5. install basic tools like wget/curl/.../autoconf/.../groff/.../build-essential/tcp-dump
6. Install X/GL build deps like libglib2.0-dev etc.
7. Install browsers: chrome and firefox
8. do gcloud config
9. clone depot_tools
10. clone buildroot

Reply:
Done!